### PR TITLE
Mobile menu: A dynamic mobile menu 

### PR DESCRIPTION
**Pull request by BB-Simon**
1. Set HTML CSS JS  linters
2. Add a mobile menu section in the HTML file
3. Select menu and close icon and run open menu and close functions

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -80,6 +80,87 @@ ul li {
...
+  top: 0;
+  left: 0;
+  width: 0%;
+  height: 100%;
+  z-index: 1;
+
+  /* Theme */
+  background: #6070ff;
+  mix-blend-mode: multiply;
+  transition: width 0.3s linear;
```
Comment (GedeonTS) [inline styles/style.css:96]: - [ ] Good job implementing a good working menu modal :clap: 
, Kindly make sure the modal's design is the same as the Figma, by  margins and paddings, and also as you can see below the background is __blur__:

Your design:
notBlur2
Figma design:

blur

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @BB-Simon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- Good use of GitHub flow :+1: 
- Good commit messages :+1: 
- Your popup menu is working perfectly :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/style.css (hunk 1 above)
Comment (BB-Simon) [reply styles/style.css:96]: Hi @GedeonTS
Thanks for your nice review, however, I changed the `background-color` blur issue, but I can see there is no `padding` or `margin` issue, please have a look.
Thanks:)

**Comment by codecaiine** [comment]
hi @BB-Simon I'm reviewing your project. I noticed that you are making some changes that could affect my review.

**Review by codecaiine — APPROVED** [review]
Hi @BB-Simon @EmekaEmmanuel @Rudy-Menekam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Great project implementation :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Menu works as expected :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**